Documentation/VSCodeConfiguration.md

```diff
@@ -130,7 +141,11 @@ These belong in the `.vscode/settings.json` of Serenity.
     "files.insertFinalNewline": true,
     // git commit message length
     "git.inputValidationLength": 72,
-    "git.inputValidationSubjectLength": 72
+    "git.inputValidationSubjectLength": 72,
+    // Tell clangd to ask the cross-compilers for their builtin include paths
+    "clangd.arguments": [
+        "--query-driver=${workspaceFolder}/Toolchain/Local/**/*"
+    ]
```

Comment:
Could you also add `"-header-insertion=never"`, clangds include style is sadly wrong in most cases

Reply:
FWIW it's not "wrong" per se. It just uses `""` for `-I` and `<>` for `-isystem`. The clangd issue is https://github.com/clangd/clangd/issues/1247 . I can add disabling it to the docs, but I don't mind having header insertion on personally. This makes me think that if we accumulate enough clangd-specific quirks and recommendations we might want a separate doc for it.